README.md

```diff
@@ -32,8 +32,8 @@ jobs:
       - name: 'Sending SMS Notification'
         uses: twilio-labs/actions-sms@v1
         with:
-          fromPhoneNumber: '+12345678900' # alternatively, use a Repository Secret
-          toPhoneNumber: '+12345678900' # alternatively, use a Repository Secret
+          fromPhoneNumber: ${{ secrets.FROM_PHONE_NUMBER }} # alternatively, use a Repository Secret
+          toPhoneNumber: ${{ secrets.TO_PHONE_NUMBER }} # alternatively, use a Repository Secret
```

Comment:
the comments from the original line don't make sense anymore since that's what we're doing